web/pdf_viewer.css

```diff
@@ -17,6 +17,7 @@
 @import url(xfa_layer_builder.css);
 /* Ignored in GECKOVIEW builds: */
 @import url(annotation_editor_layer_builder.css);
+@import url(draw_layer_builder.css);
```

Comment:
I'm assuming that you don't, currently, want this in the GeckoView-specific viewer?
If so, there's two possible solutions:
 - Place this import at the top of the `annotation_editor_layer_builder.css` file instead (this is probably the nicest solution).
 - Add the new CSS file to the list in https://github.com/mozilla/pdf.js/blob/f4b396f6c88e951174879a52390fa89223326b36/external/builder/builder.mjs#L45-L49